services/frontend-v3/src/components/layouts/aboutLayout/AboutLayoutView.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import { Typography, Tabs, Row, Result, Button } from "antd";
+import { Redirect } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import { SettingTwoTone } from "@ant-design/icons";
+import AppLayout from "../appLayout/AppLayout";
+
+const { TabPane } = Tabs;
+const { Title } = Typography;
+let numTab = "1";
+/**
+ *  AboutLayoutView(props)
+ *
+ *  this component renders the about page.
+ */
+
+const styles = {
+  outerRow: { marginTop: "20px" },
+  outerTabs: {
+    width: "95%",
+    backgroundColor: "#ffffff",
+    padding: "20px",
+    height: "90%",
+  },
+  titles: {
+    paddingTop: "20px",
+    textAlign: "center",
+  },
+};
+
+// this function checks the end of the URL for either about, privacy, or terms, to determine what
+// the default tab will be on the about page
+
+function checkURL() {
+  const urlSections = window.location.toString().split("/");
```

Comment:
Instead of relying on the URL to extract which component, we should probably just pass the type (either "help", "terms", "privacy", and "about"). By doing so, we wouldn't need this function